### PR TITLE
Fix indentation, consistent usage of string quotes

### PR DESCRIPTION
**Pull request by dbro86**
Fix indentation, consistent usage of string quotes

### CODE REVIEW

**Comment by tcolgate** [comment]
Sorry, I feel a bit mean, but I'm closing this largely because I just don't like the alternative alignment. If the internet settled on a standard yamlfmt while I wasn't looking, let me know and I'll reconsider.

**Comment by dbro86** [comment]
I don't understand your explanation.
The indentation should be the same over the whole file.
This file contains an inventation sometimes of 3 spaces, sometimes of 2 spaces.
Furthermore sometimes you use single string quotes, sometimes double string quotes, and sometimes do not use string quotes.
